third-party/CMakeLists.txt

```diff
@@ -21,22 +21,32 @@ message("Entering ${CMAKE_CURRENT_SOURCE_DIR}")
 
 option(USE_SYSTEM_CLI11 "Use the CLI11 lib of the operating system" OFF)
 if(USE_SYSTEM_CLI11)
-  find_package(CLI11 "2.2.0" CONFIG REQUIRED)
-  set_target_properties(CLI11::CLI11 PROPERTIES IMPORTED_GLOBAL TRUE)
+  find_package(CLI11 "2.4.2" CONFIG REQUIRED)
+#  set_target_properties(CLI11::CLI11 PROPERTIES IMPORTED_GLOBAL TRUE)
   message(STATUS "Using system CLI11 ${CLI11_VERSION}")
-else()
-  add_subdirectory(CLI11 EXCLUDE_FROM_ALL)
+# else()
+#  add_subdirectory(CLI11 EXCLUDE_FROM_ALL)
```

Comment:
I would expect the CPMAddPackage calls to happen here in the else block.  Do you know why we did not do it this way ?

Reply:
I've redone the workflow, so those are now place into the `cmake/BareosCpmPackages.cmake`